assets/src/edit-story/elements/video/karma/autoplay.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { getBackgroundElementId } from '../../../components/dropTargets/karma/background.karma';
+
+fdescribe('Autoplay video', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should render ok', () => {
+    expect(
+      fixture.container.querySelector('[data-testid="fullbleed"]')
+    ).toBeTruthy();
+  });
+
+  it('should autoplay on insert and on drop', async () => {
+    // TODO: Switch to role selector after they are merged
+    const videoFilter = fixture.screen.getByText('Video', {
+      exact: true,
+      selector: 'button',
+    });
+    await fixture.events.mouse.clickOn(videoFilter);
+    const videos = fixture.screen.getAllByTestId('mediaElement');
```

Comment:
Not sure how we want to utilize `@testing-library` but that's one easy way. @dvoytenko @barklund

Reply:
Give the video element `role="application"` ([the only allowed role for video](https://www.w3.org/TR/html50/embedded-content-0.html#the-video-element)) and put the `alt` attribute from the media query on the video element as `aria-label`.

Then you should be able to find it by `getByRole('application', { name: 'alt value' });`.

I'm not sure if it has any negative consequence to put this role on the element though.

Reply:
But other than this, this is a nice and simple way to introduce the selectors from RTL. Do we need to pre-configure it with `container` as the target or is there nothing else to target?

Reply:
I didn't find any relevant information about video element a11y, in practice, they mostly don't have roles assigned. So adding that might be unnecessary, but I did it and we will see how it plays out.

Screen:
We should be good with it, even with direct import from RTL. I just added that to fixture in case we want to do something extra there. No additional setup is needed.
BTW: more on screen and container [here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library).